src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(101), ColorSupplier.getRandomColor());
```

Comment:
make Random and ColorSupplier like a class Fields
101 is magic number, move this to the class constant

Reply:
extracted to variable